class.jetpack-xmlrpc-server.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+use Automattic\Jetpack\Tracking\Manager as JetpackTracking;
```

Comment:
If we're going to keep the legacy name, probably best to classmap the legacy class.

Otherwise, change it to the new name everywhere.